clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -289,41 +288,191 @@ public void testAppend() throws IOException {
         fs.append(null, 0, null);
     }
 
-    /*
+    private void mockNonExistingPath(ObjectLocation objectLoc) throws ApiException {
+        when(objectsApi.statObject(objectLoc.getRepository(), objectLoc.getRef(), objectLoc.getPath()))
+                .thenThrow(new ApiException(HttpStatus.SC_NOT_FOUND, "no such file"));
+        when(objectsApi.listObjects(objectLoc.getRepository(), objectLoc.getRef(),
+                objectLoc.getPath() + Constants.SEPARATOR, "", 1, ""))
+                .thenReturn(new ObjectStatsList());
+    }
+
+    private void mockExistingDirPath(ObjectLocation dirObjLoc, ObjectLocation fileInDir) throws ApiException {
```

Comment:
Please document this function, it is unclear to a reader like myself what the parameters mean & what it does.